code/controllers/subsystem/mapping.dm

```diff
@@ -46,6 +48,8 @@ SUBSYSTEM_DEF(mapping)
 
 
 /datum/controller/subsystem/mapping/Initialize(timeofday)
+	if(initialized)
```

Comment:
Is this necessary? I should hope the MC doesn't double-initialize us.

Reply:
Only time this would run is during a hard mc crash, it's trying to recreate the game from scratch. If you need to handle that, use Recover() instead

Reply:
@SpaceManiac if it initializes again it'll try to load in every zlevel on top of the existing zlevels which would be really bad, believe it's the only subsystem that will immediately cause a server crash.
why i did this: i decided to proccall initialize locally.

Reply:
There are plenty options for accidentally fucking up everything you don't need to start protecting against admin stupidity here.

Reply:
Stop trying to prevent crashes from vv/sdql. The only thing you should be preventing is privilege escalation, shell/call access, reading/writing protected configs, and filesystem access.

Reply:
I'm not worried about having this line. Compared to most other subsystems, doubling the number of z-levels in case of some freak MC accident will definitely crash out. If anyone wants to remove this later they can.